pavelib/paver_tests/test_eslint.py

```diff
@@ -9,13 +9,13 @@
 import pavelib.quality
 
 
-class TestPaverESLint(unittest.TestCase):
+class TestPaverStylelint(unittest.TestCase):
```

Comment:
I'm not clear on what you wanted here.  Class went from `ESLint` to `Stylelint`, but this is in `test_eslint.py`?  Should these changes be reverted?

Reply:
Whoops. Reverting now.